src/rgbds.5

```diff
@@ -127,6 +128,18 @@ REPT NumberOfSections
     ENDC
 
 ENDR
+
+; Assertions
+
+REPT NumberOfAssertions
+    STRING  ErrorMessage
+    STRING  SourceFile
+    LONG    Line
+    BYTE    Type ; 0 = assert
+                 ; 1 = warnif
```

Comment:
I think it would be better if this agreed with the enum you added, to avoid confusion.
```
enum AssertSeverity {
	SEV_WARN = 0,
	SEV_FAIL = 1,
};
```
i.e. something like this:
```
    BYTE    Severity ; 0 = warn
                     ; 1 = fail
```